.github/workflows/benchmark.yml

```diff
@@ -3,25 +3,26 @@ name: Benchmarking tests
 on:
   workflow_dispatch:
   schedule:
-    - cron: "30 1 1,15 * *" # every 2 weeks on the 1st and the 15th of every month at 1:30 AM
+    - cron: "0 0 * * *" # every day at midnight
 
 env:
   DIFFUSERS_IS_CI: yes
   HF_HUB_ENABLE_HF_TRANSFER: 1
   HF_HOME: /mnt/cache
   OMP_NUM_THREADS: 8
   MKL_NUM_THREADS: 8
+  BASE_PATH: benchmark_outputs
 
 jobs:
-  torch_pipelines_cuda_benchmark_tests:
+  torch_models_cuda_benchmark_tests:
     env:
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL_BENCHMARK }}
-    name: Torch Core Pipelines CUDA Benchmarking Tests
+    name: Torch Core Models CUDA Benchmarking Tests
     strategy:
       fail-fast: false
       max-parallel: 1
     runs-on:
-      group: aws-g6-4xlarge-plus
+      group: aws-g6e-xlarge-plus
```

Comment:
Can be done in a follow up PR but it would be good to run a benchmark for more resource constrained environments too with a small GPU.

Reply:
Indeed. Some tests might need a bigger GPU and I think we can catch and report them appropriately. Do you want to take a stab at this in a follow-up?